bin/docker-psql

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+docker exec -it `docker ps | grep sentry_postgres | awk '{print $1}'` bash -c "psql -U postgres sentry"
```

Comment:
We can also safely assume the name is `sentry_postgres`. And you don't need the additional `bash -c` here.

I'd rather we just do this in Python if we want to expose this over a `sentry <cmd>` since we alreayd require `docker-py`, there's no reason not to just use that and avoid this shim script.